server/routes/journeys/update-decision/conclusion/view.njk

```diff
@@ -0,0 +1,34 @@
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "partials/submitButton/macro.njk" import submitButton %}
+
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = "Conclusion - Update a CSIP investigation decision" %}
+
+{% block innerContent %}
+    {% include 'base/components/decision-caption.njk' %}
+    <label for="conclusion">
+        <h1 class="govuk-heading-l govuk-!-margin-bottom-7">Add information about the reasons for the decision</h1>
+    </label>
+    <form method="post">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+        {{ govukInsetText({
+            html: currentConclusion | escape | boldAppendStamp | nl2br
+        }) }}
```

Comment:
currentConclusion can potentially be nullish. and insettext component shouldn't be shown if so.

Reply:
Fixed